.azure-pipelines/templates/automation_test.yml

```diff
@@ -13,6 +13,8 @@ steps:
 
       source env/bin/activate
 
+      pip install git+https://github.com/haroldrandom/azure-python-devtools.git@fix-python-3.8.3
```

Comment:
Will revert it once https://github.com/Azure/azure-python-devtools/pull/65 is merged